lib/puppet/provider/pulp_rpmbind/consumer.rb

```diff
@@ -0,0 +1,41 @@
+Puppet::Type.type(:pulp_rpmbind).provide(:consumer) do
+  desc 'Bind/unbind to an RPM repo'
+
+  confine osfamily: :redhat
+  commands consumer: '/bin/pulp-consumer'
+  commands grep:     '/bin/grep'
+
+  def self.instances
+    begin
+      binds = grep('-oP', '^\[\K.*(?=\]$)', '/etc/yum.repos.d/pulp.repo')
```

Comment:
Done!  https://github.com/Katello/puppet-pulp/pull/231/commits/ad4d4a2b5147686f06f18b22fba39baf22742cdc

@ekohl Let me know what you think.  If you're happy with the change, I'll squash the commits.